src/utils/shared.test.ts

```diff
@@ -13,6 +13,22 @@ describe("getChainId for v2 document", () => {
     expect(getChainId(document)).toStrictEqual(5);
   });
 
+  it("should return the correct chainId for polygon mumbai network", () => {
```

Comment:
statement didn't tally with what was tested

Reply:
how? the test is just reading the file that was provided and retrieve the chainId only arh.